scss/bootstrap/_variables.scss

```
@@ -828,14 +828,12 @@ $dropdown-link-active-bg: $component-active-bg !default;
 
 $dropdown-link-disabled-color: $gray-600 !default;
 
-$dropdown-item-padding-y: 0.5rem;
-$dropdown-item-padding-x: 1.5rem !default;
+$dropdown-item-padding-y: $spacer * 0.25 !default;
```

```suggestion
$dropdown-item-padding-y: $spacer * 0.25;
```

Pas sur de comprendre pk essayer de se baser sur le spacer ici